### PR TITLE
#147 Bom functions

### PR DESCRIPTION
**Pull request by eLeontev**
Issue: #147
Changes:
1. add description for next BOM objects:
  - screen
  - location
2. Location fields are described. Screen fields are not described due to their legacy roots (rarely used for now)
3. Doc file is created

### CODE REVIEW

File: JavaScript_Advance/BOM_functions.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  colorDepth,
+  height,
+  orientation,
+  pixelDepth,
+  width
+} = window.screen;
+
+// window location is object provides interface and API to handle client url
+const {
+  assign,    /** method to manual switch to another url location.assign(path_to_page) */
```
Comment (Swap76) [inline JavaScript_Advance/BOM_functions.js]: @eLeontev the assign is blue but why not all other things below it

File: docs/JavaScript_Advance/BOM_functions.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# BOM objects: screen / location
```
Comment (Swap76) [inline docs/JavaScript_Advance/BOM_functions.md]: Need info here

File: JavaScript_Advance/BOM_functions.js (hunk 1 above)
Comment (eLeontev) [reply JavaScript_Advance/BOM_functions.js]: looks like here is issue with comments view, let me update it tomorrow with jsDoc format, also I will ad the similar info in doc file.

File: docs/JavaScript_Advance/BOM_functions.md (hunk 2 above)
Comment (eLeontev) [reply docs/JavaScript_Advance/BOM_functions.md]: done

**Comment by Swap76** [comment]
@eLeontev It was great working with you.

Looking forward to working with you :-)

Star to the repo is highly appreciated.

**Comment by Swap76** [comment]
@eLeontev The work you have done for the project is highly appreciated. Thanks :-)

Looking forward to working with you on my new project related to Deno https://github.com/Swap76/Learn-Deno/ and together making this better.

Star to the repo is highly appreciated.

**Pull request merged**